File: BaekJoon_python/2480.py
A, B, C = map(int, input().split())
answer = 0

if(A == B and B == C) : answer = 10000+A*1000
elif (A == B and B != C) : answer = 1000+A*100
elif (A == C and C != B) : answer = 1000+C*100
elif (B == C and A != B) : answer = 1000+B*100
else : answer = max(A, B,C) * 100

print(answer)

File: js/BaekJoon/2480.js
const fs = require("fs");

//const input = fs.readFileSync("input.txt").toString().trim().split(" ").map(Number);
const input = fs
  .readFileSync("/dev/stdin")
  .toString()
  .trim()
  .split(" ")
  .map(Number);

const [a, b, c] = [input[0], input[1], input[2]];

let answer;

if (a == b && b == c) {
  answer = 10000 + a * 1000;
} else if (a == b && b != c) {
  answer = 1000 + a * 100;
} else if (a == c && c != b) {
  answer = 1000 + c * 100;
} else if (b == c && a != b) {
  answer = 1000 + b * 100;
} else {
  answer = Math.max(a, b, c) * 100;
}

console.log(answer);
